theano/compile/tests/test_function_module.py

```diff
@@ -60,79 +63,88 @@ def test_missing_inputs(self):
         def fn():
             x, s = T.scalars('xs')
             fn = function([], [x])
+            return fn
         checkfor(self, fn, MissingInputError)
```

Comment:
After a discussion with @lamblin on this, it is better not to return anything here as we just want to check that the function call raises an error. The way to remove the flake8 error is then not to assign the function to anything, i.e.
instead of this:
``` python
fn = function([], [x])
return fn
```
having this:
``` python
function([], [x])
```

Please also update the subsequent functions accordingly.